mock/py/mockbuild/config.py

```diff
@@ -258,6 +258,7 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
     config_opts['macros'] = {
         '%_topdir': '%s/build' % config_opts['chroothome'],
         '%_rpmfilename': '%%{NAME}-%%{VERSION}-%%{RELEASE}.%%{ARCH}.rpm',
+        '%_platform_multiplier': 1,
```

Comment:
Don't we want to use something with `%mock_` prefix?  I'd personally think this is somet default variable used by rpmbuild.

Reply:
It can be. I wanted to raise it for rpm.org or Fedora Packaging Guidelines. As I can imagine having this macro defined on system-level and e.g. slow armhpf can have this set to 2 by default.